core/modules/drr.cc

```diff
@@ -274,21 +303,21 @@ void DRR::AddNewFlow(bess::Packet* pkt, FlowId id, int* err) {
   Flow* f = new Flow(id);
 
   // TODO(joshua) do proper error checking
-  f->queue = AddQueue(static_cast<int>(kFlowQueueSize), err);
+  f->queue = generator_();
```

Comment:
Given that there are a fixed number of flows, I wonder if it might be worth considering preallocating these queues, avoiding some of the initial allocation costs.